source/time_tables/thermometer.cpp

```diff
@@ -140,7 +140,7 @@ struct VertexProperties
     VertexProperties() : idx(0) {}
     VertexProperties(type::idx_t const& i): idx(i) {}
 };
-using Graph = boost::adjacency_list<boost::vecS, boost::vecS, boost::directedS, boost::property<boost::vertex_bundle_t, VertexProperties> >;
+using Graph = boost::adjacency_list<boost::vecS, boost::vecS, boost::directedS, VertexProperties>;
```

Comment:
@xavierraffin we are going to merge that because it just did not compile on clang 3.6.0 and gcc 4.9.2, and tests are ok, but if there was a special purpose for that use of vertex_bundle we will have to do something else...